core-java-modules/core-java-arrays-guides/pom.xml

```diff
@@ -24,6 +24,13 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <version>${jmh-generator.version}</version>
         </dependency>
+        <dependency>
+            <groupId>uk.org.webcompere</groupId>
+            <artifactId>system-stubs-jupiter</artifactId>
+            <version>2.1.5</version>
```

Comment:
We have a convention to extract a property for dependency versions, as you can see with the above dependencies

Reply:
@Neetika23 this was not resolved

Reply:
Code working now. I have checked the dependencies.